src/tree/test_treemath.rs

```diff
@@ -54,7 +54,7 @@ fn test_tree_hash() {
 
     fn create_identity(id: &[u8], ciphersuite: &Ciphersuite) -> KeyPackageBundle {
```

Comment:
If this is consuming `ciphersuite`, it shouldn't get a reference.

Reply:
Let's leave the whole identity stuff for a separate PR